### PR TITLE
Release: Update payloads for v0.9.0-alpha1

### PR DESCRIPTION
**Pull request by portersrc**
This updates the payloads for v0.9.0-alpha1

### CODE REVIEW

**Comment by fitzthum** [comment]
We have an issue with the enclave-cc test. This looks somewhat similar to the issue we saw with the last release, which had to do with the yaml files in the test being out of sync. Enclave-cc has had very few changes since that was fixed. We fiddled with the sgx CI node some, but I think this test is not running there. Not sure what the cause is yet.

cc @Xynnn007

**Review by fidencio — APPROVED** [review]
lgtm, thanks @portersrc!

I'd like to bother you, though, to keep this as a draft till we have the enclave-cc issue fixed.  So it avoids us merging it before running that CI.

File: config/samples/enclave-cc/base/ccruntime-enclave-cc.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ spec:
       node.kubernetes.io/worker: ""
   config:
     installType: bundle
-    payloadImage: quay.io/confidential-containers/runtime-payload-ci:enclave-cc-HW-cc-kbc-latest
+    payloadImage: quay.io/confidential-containers/runtime-payload-ci:enclave-cc-HW-cc-kbc-v0.9.1
```
Comment (fitzthum) [inline config/samples/enclave-cc/base/ccruntime-enclave-cc.yaml:12]: I think this needs to change from `runtime-payload-ci` to `runtime-payload`

Comment (portersrc) [reply config/samples/enclave-cc/base/ccruntime-enclave-cc.yaml:12]: ohh great catch; thank you

Comment (fidencio) [reply config/samples/enclave-cc/base/ccruntime-enclave-cc.yaml]: Very nice catch, indeed!

```
@github-actions
enclave-cc e2e tests / operator enclave-cc e2e tests (pull_request) Successful in 5m
```

**Review by fitzthum — APPROVED** [review]
LGTM

**Comment by fidencio** [comment]
Please, do not wait for the TDX machine for merging it.

I took the machine down Today as I was trying to properly deal with some leftovers that were causing issues with re-runs, and now I'm in the middle of a whac-a-mole regarding Ubuntu 24.04 support for the ansible stuff.

**Comment by fitzthum** [comment]
I am going to wait until tomorrow to tag the repo to make sure that we don't need something else for CAA.

cc @stevenhorsman

**Comment by stevenhorsman** [comment]
> I am going to wait until tomorrow to tag the repo to make sure that we don't need something else for CAA.
> 
> cc @stevenhorsman

I think you are fine to go ahead this release, but we should update the checklist to have the peer pods stage before the operator if we are still planning to stick with the previously discussed plan?

**Pull request merged**